examples/cfg.yaml

```diff
@@ -1,93 +1,94 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/alex-courtis/way-displays/refs/heads/master/schema/cfg-1.0.0.yaml
 
-# Default cfg.yaml for way-displays.
-# Copy this to ~/.config/way-displays/cfg.yaml and edit it to your liking.
-#
-# See https://github.com/alex-courtis/way-displays/wiki/Configuration
+## Default cfg.yaml for way-displays.
+## Copy this to ~/.config/way-displays/cfg.yaml and edit it to your liking.
+
+## See https://github.com/alex-courtis/way-displays/wiki/Configuration
 
 
-# Arrange displays in a ROW (default, left to right) or a COLUMN (top to bottom)
+## Arrange displays in a ROW (default, left to right) or a COLUMN (top to bottom)
 ARRANGE: ROW
 
-# Align ROWs at the TOP (default), MIDDLE or BOTTOM
-# Align COLUMNs at the LEFT (default), MIDDLE or RIGHT
+## Align ROWs at the TOP (default), MIDDLE or BOTTOM
+## Align COLUMNs at the LEFT (default), MIDDLE or RIGHT
 ALIGN: TOP
 
 
-# The default ORDER is simply the order in which the displays are discovered.
-# Define your own.
-ORDER:
-  # - 'DP-2'
-  # - 'monitor description'
-  # - 'HDMI-1'
+## The default ORDER is simply the order in which the displays are discovered.
+## Define your own.
+# ORDER:
```

Comment:
Commenting out the entire element for correctness.

That seems simpler than using an empty list, as uncommenting the examples results in bad yaml.

Existing users should be fine; if they use a language server they'll be savvy enough to know why it doesn't validate.